makefile

```diff
@@ -47,13 +47,34 @@ ifneq (,$(PREFIX))
 STACK_BIN_PATH := --local-bin-path $(PREFIX)
 endif
 
+possible-clang-locations := clang++-9 clang++-10 clang++-11 clang++
+
+CLANG := $(shell if [[ "$$DEX_LLVM_HEAD" -eq 1 ]] ; \
+	then echo "clang++"; \
+	else \
+		for clangversion in $(possible-clang-locations) ; do \
+		if [[ $$(command -v "$$clangversion" 2>/dev/null) ]] ; \
+		then echo "$$clangversion" ; break ; \
+		fi ; \
+		done ; \
+	fi)
```

Comment:
Instead of putting the conditional on DEX_LLVM_HEAD couldn't we set `CLANG` to be `clang++` by default and override its value using the `for` loop you have here in the `else` case of the `ifeq (1,$(DEX_LLVM_HEAD))` conditional?